mender-update/update_module/v3/update_module.cpp

```diff
@@ -51,9 +51,9 @@ std::string StateToString(State state) {
 
 UpdateModule::UpdateModule(MenderContext &ctx, const string &payload_type) :
 	ctx_(ctx) {
-	update_module_path_ = path::Join(conf::paths::DefaultModulesPath, payload_type);
+	update_module_path_ = path::Join(ctx.modules_path, payload_type);
 	update_module_workdir_ =
-		path::Join(conf::paths::DefaultModulesWorkPath, "payloads", "0000", "tree");
+		path::Join(ctx.modules_work_path, "modules", "v3", "payloads", "0000", "tree");
```

Comment:
Q: Why change this? I feel like the default paths read a lot better?

Reply:
Yes, but the tests need to be able to change it using the `ctx`. While we could change the global default, I think it is better to keep changes localized, even if readability suffers a little.

Reply:
Hmm, how exactly are we changing them in the tests? 
During runtime? :thinking:

Reply:
Yes, they are being changed [here](https://github.com/mendersoftware/mender/pull/1252/files#diff-2fdfef8757e1cd478baf77bc0e427559352897984a53f53c18825b17ea3758fcR236) by most of the tests. Otherwise it tries to look in `/usr/share/mender`, which won't work while testing.